app/src/main/java/com/fs_sournary/weather/screen/main/fragment/current/CurrentWeatherFragment.java

```diff
@@ -7,6 +7,7 @@
 import android.view.View;
 import android.view.ViewGroup;
 
+import com.fs_sournary.weather.BuildConfig;
```

Comment:
ERROR: Unused import: com.fs_sournary.weather.BuildConfig. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports